src/map/packet_system.cpp

```diff
@@ -2163,18 +2163,33 @@ void SmallPacket0x04E(map_session_data_t* session, CCharEntity* PChar, CBasicPac
             !(PItem->isSubType(ITEM_LOCKED)) &&
             !(PItem->getFlag() & ITEM_FLAG_NOAUCTION))
         {
-            if (quantity == 0 &&
-                (PItem->getStackSize() == 1 ||
-                PItem->getStackSize() != PItem->getQuantity()))
+            uint32 AUCTION_FEE = 0;
```

Comment:
don't use caps for a local variable (caps are usually reserved for macros and enums)